troubleshooting.md

```diff
@@ -1110,50 +1135,51 @@ ls: cannot open directory '/dir1': Permission denied
 
 We follow essentially the same solution as in the previous
 troubleshooting tip:
- "_Container creates a file that is not owned by the regular UID_"
+
+    Container creates a file that is not owned by the regular UID
+	
```

Comment:
White space leak is here

Reply:
it was a tab, that's why searching for "\ +$" in vim didn't find it. thanks!